### ISSUE TITLE
Old Champion's Remains cause server crash.

### ISSUE THREAD

**Issue by 5056XD** [mariumbacchus/Soulslike-Weaponry#93]
When the survival mode player hit the Old champion remains, the server will crash immediately due to ticking entity (which wrote on crash report). If the player rejoin, it will crash again the server. Only can be join again by keep sending creative mode commands on server console. Even so, the player mod's item will gone (including placed item) except the one in inventory. Other player doesn't affected.

![Screenshot 2024-02-16 013650](https://github.com/mariumbacchus/Soulslike-Weaponry/assets/139415904/7d93f932-7c4a-453b-a5e0-2c76d53322a1)
![Screenshot 2024-02-16 012638](https://github.com/mariumbacchus/Soulslike-Weaponry/assets/139415904/b5ad4afd-04eb-4780-b02f-119d7b2a158c)
[crash-2024-02-16_00.48.44-server.txt](https://github.com/mariumbacchus/Soulslike-Weaponry/files/14300640/crash-2024-02-16_00.48.44-server.txt)

**Comment by mariumbacchus** [comment]
Is this a consistent issue? Also, can you rephrase it? Is the issue the crash itself or did it cause other modded items to disappear? Couldn't really understand the last part about items disappearing. It looks like some mods were missing on the server side or something.

**Comment by 5056XD** [comment]
> Is this a consistent issue? Also, can you rephrase it? Is the issue the crash itself or did it cause other modded items to disappear? Couldn't really understand the last part about items disappearing. It looks like some mods were missing on the server side or something.

The issue is the crash itself. Every time the old champion use attack the server will crash (not client). The only way to solved it, is make the player near the boss creative mode by keep sending the command on server console when the player join. But still cause affected player's chest mods item, mods block that placed disappear only (other player which not participate was not affected).

**Comment by mariumbacchus** [comment]
That might be it. I have not made a version which fits 1.20.2 perfectly yet, and mojang really loves to randomly change the code between smaller updates. I'll make a version to fit the newest version of minecraft after the next mod update.

**Comment by Ramst3r** [comment]
As of today I encountered this same ticking entity issue, only when being killed by the champion. I'm on 1.20.1 fabric. The log seems to mention a possible bug with spell engine.
[latest.log](https://github.com/mariumbacchus/Soulslike-Weaponry/files/14338729/latest.log)

**Comment by mariumbacchus** [comment]
Yes, this seems to be a spell engine issue. Not sure what it means, you should set up an issue over there and see if they can fix it. If this truly is a Soulslike Weaponry issue, then you can make a new issue here and I'll try to help then.
